include/geode/mesh/core/surface_mesh.h

```diff
@@ -328,7 +328,8 @@ namespace geode
          * @param[in] vertex_id Index of the vertex.
          * @pre This function needs that polygon adjacencies are computed
          */
-        PolygonsAroundVertex polygons_around_vertex( index_t vertex_id ) const;
+        const PolygonsAroundVertex& polygons_around_vertex(

```

Comment:
You should check when it is used to be sure we use correctly the ref

Reply:
I did it in OG. By default it is fine because it was a return by value, so worst case we do a copy instead of a ref.